app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/ProblemWithAddressController.scala

```diff
@@ -31,6 +31,7 @@ class ProblemWithAddressController @Inject() (
     extends FrontendController(cc) {
 
   def problem(journey: JourneyBindable): Action[AnyContent] = Action { implicit request =>
-    Ok(problemWithAddressPage(journey))
+    val postAction: Call = routes.CheckContactDetailsMrnController.changeAddress(journey)
```

Comment:
Optional: postAction is only used once so you can just inline it

Reply:
I could, but I'm being pedantic ;)